tiledb/bioimg/visualization/helpers.py

```diff
@@ -0,0 +1,65 @@
+import json
+from dataclasses import dataclass
+from typing import List, Optional, Sequence
+
+import tiledb
+
+from ..converters.axes import Axes
```

Comment:
Will there be (a lot) more stuff under `visualization` in the near to mid-term future? If not I'd keep it in a `visualization.py` module until it grows enough to justify a subpackage. 

As an aside, I think the `converters` subpackage was a bad call in retrospect given the coupling between the modules: IMO `from ..converters.axes import Axes` is a code structure smell, and so are these:
```py
# tiledb.bioimg.openslide
from .converters.axes import Axes

# tiledb.bioimg.converters.base
from ..helpers import ...
from ..openslide import ...
```